src/definitions/modules/calendar.js

```diff
@@ -141,12 +141,16 @@ $.fn.calendar = function(parameters) {
               $container = $('<div/>').addClass(className.popup)[domPositionFunction]($activatorParent);
             }
             $container.addClass(className.calendar);
-            var onVisible = settings.onVisible;
+            var onVisible = function () {
+              module.refreshTooltips();
+              return settings.onVisible.apply($container, arguments);
```

Comment:
Why `function.apply`?
It seems that the passed `$container` (used as `this` of `onVisible` function) is never used inside `onVisible`.

Reply:
But it **could**, because the `onVisible` method is a dynamic setting which can be adjusted by the user. It's already used this way everywhere else in the code. Also the docs tell, that the context is the active content

![image](https://user-images.githubusercontent.com/18379884/87912120-102f4200-ca6d-11ea-9284-08ea26d6de74.png)

Reply:
Ah, sorry my bad...
I overlooked that it is `settings.onVisible`, not `onVisible` created in this code.